Do not fail if related usns fail (SC-1300) (#2497)
canonical/ubuntu-pro-client: docs/references/what_is_a_cve.md

# What is a CVE

> A CVE (Common Vulnerabilities and Exposures) is a way to catalog and track public security

Reviewer: ```suggestion
Common Vulnerabilities and Exposures (CVEs) are a way to catalogue and track public security
```

Reviewer: It's generally better to put acronyms and their expansions this way around so that you're introducing the concept first (and then the name you're going to call it by)

Reviewer: Also, should we include this link? https://ubuntu.com/security/cves

Author: Good point, I will add a paragraph for that

vulnerabilities for a given software. Every CVE is identified through a unique identifier,
for example [CVE-2023-0465](https://www.cve.org/CVERecord?id=CVE-2023-0465).

> CVEs are maintained by the [MITRE Corporarion](https://cve.mitre.org/) and the goal of the project

Reviewer: ```suggestion
CVEs are maintained by the [MITRE Corporation](https://cve.mitre.org/) and the goal of the project
```

is to provide naming conventions for the public known security issues while also maintaining a

> centralized repository for all of the security issues. This makes it easier for a organization to

Reviewer: ```suggestion
centralised repository for all of the security issues. This makes it easier for an organization to
```

> submit a new security flaw thought the CVE convention while also analyzing any other existing CVEs

Reviewer: ```suggestion
submit a new security flaw though the CVE convention while also analysing any other existing CVEs
```
